src/components/MoneyInput.svelte

```diff
@@ -7,9 +7,10 @@ import { onMount } from 'svelte'
 export let label = ''
 export let value = ''
 export let placeholder = ''
-export let maxlength: number = 524288 /* default */
+export let maxlength: number = 2048 /* default */
```

Comment:
I guess this should be 255 as well.
```suggestion
export let maxlength: number = 255/* default */
```